announcements/move-to-cluster-api.md

```diff
@@ -0,0 +1,42 @@
+### What is Cluster API?
+
+Cluster API is a Kubernetes sub-project focused on providing declarative APIs and tooling to simplify provisioning, upgrading, and operating multiple Kubernetes clusters.
+
+Started by the Kubernetes Special Interest Group (SIG) Cluster Lifecycle, the Cluster API project uses Kubernetes-style APIs and patterns to automate cluster lifecycle management for platform operators. The supporting infrastructure, like virtual machines, networks, load balancers, and VPCs, as well as the Kubernetes cluster configuration are all defined in the same way that application developers operate deploying and managing their workloads. This enables consistent and repeatable cluster deployments across a wide variety of infrastructure environments. The project was started over 2 years ago, and is growing and improving rapidly. You can find more details [here](https://cluster-api.sigs.k8s.io/).
+
+### Why is Giant Swarm moving to Cluster API?
+
+In 2017, due to a lack of better alternatives, Giant Swarm decided to build their product based on their own implementation of automation, tools, and operators. This allowed Giant Swarm to provide a stable and easy-to-use product that you are currently using to manage your Kubernetes clusters. Although this solution was ideal for a long time it has its downsides:
+- You have limited control over configuration of individual parts of the infrastructure. The product is not flexible enough and sometimes doesn't support all of your use cases.
+- Adding new functionality is time-consuming and as hyperscalers (Amazon, Microsoft) expand their offering it becomes increasingly difficult to keep up and make it all available to you in a timely manner.
+- Your cluster management is limited to what Giant Swarm tooling enables and there are no available alternatives.
+- Although Giant Swarm provides pure "vanilla" Kubernetes clusters, it is currently not possible to move the workload clusters and have them managed by another product without migration. 
+
+Giant Swarm has been closely watching the development of the upstream Cluster API project and since it achieved a satisfying level of maturity, decided to adopt it in order to address all of the downsides mentioned above and bring more value to you:
```

Comment:
> achieved a satisfying level of maturity

I thought about this and I don't wanna bash upstream at all here, but I think it's rather that we think the timing is right because the level of maturity is "just enough" for us to be able to make it work for customers, while at the same time us going all-in on this now will be a good use of our time contributing to increasing maturity upstream and also bringing in our experience of many years of production operations with our current product early enough for it to have an impact.

Reply:
it's a balance between going in too early when reliability might be low and going in too late when our input/contribution would be too late

Reply:
I was thinking that calling it a "satisfying level of maturity" might be a little more reassuring for some customers than "just enough" - and aren't those wordings close enough? Please let me know if you have an idea how to phrase this better.

Reply:
true, close enough, nit picky almost ;)